mars/services/web/ui/src/task_info/TaskListPage.js

```diff
@@ -84,6 +86,11 @@ class TaskList extends React.Component {
                             <TableCell>{task['end_time'] ? formatTime(task['end_time']) : 'N/A'}</TableCell>
                             <TableCell>{Math.floor(task['progress'] * 100).toString() + "%"}</TableCell>
                             <TableCell>{this.formatTaskStatus(task)}</TableCell>
+                            <TableCell>
```

Comment:
I'd like to add a `Task Detail` page and put task visualization inside it as a tab page, just like other server objects.

Reply:
New Web UI:

a Task Detail tab on left menu:

<img width="1440" alt="Screen Shot 2021-08-05 at 8 35 31 PM" src="https://user-images.githubusercontent.com/69881587/128350785-c2f377d5-7c1a-4664-85b7-eb820c543ce0.png">

when user clicks, a list of all the task ids will be displayed:

<img width="1440" alt="Screen Shot 2021-08-05 at 8 36 20 PM" src="https://user-images.githubusercontent.com/69881587/128350876-559b570f-f157-4d64-88f3-ecb2d587e71f.png">

Clicking on one of the ids will bring the user to the visualization page:

<img width="1440" alt="Screen Shot 2021-08-05 at 8 36 56 PM" src="https://user-images.githubusercontent.com/69881587/128350952-ff883a80-8397-4bd1-ab15-d249466f32ab.png">